lib/inputstreamhelper.py

```diff
@@ -449,6 +449,41 @@ def _supports_widevine(self):
 
         return True
 
+    def _select_best_chromeos_image(self, devices):
+        best = None
+        for device in devices:
+            # Select ARM hardware only
+            for arm_hwid in config.CHROMEOS_RECOVERY_ARM_HWIDS:
+                if arm_hwid in device['hwidmatch']:
+                    hwid = arm_hwid
+                    break  # We found and ARM device
```

Comment:
Typo in comment?

Reply:
Yes, still not infallible yet. Working on that ;-)

Reply:
Fixed.